core-feature-pack/common/src/main/resources/content/bin/systemd/README

```diff
@@ -0,0 +1,80 @@
+= How to configure WildFly as a Systemd service
+
+The following steps describe the process to configure WildFly as a Systemd service.
+You can use the provided wildfly-[standalone,domain].service file as a template for your own Systemd Unit File.
+It can be adjusted to your needs, for example, you can manually change the user that runs the service,
+the location of the WildFly installation, logs, etc.
+Alternatively, you can use the generate_systemd_unit.sh script to automatically generate a new Systemd Unit File
+using this server installation as the WildFly home.
+
+If you want to install WildFly as a Systemd service from scratch as a Systemd service, follow the steps below.
+
+== Install WildFly
+
+    # unzip -d /opt wildfly-34.0.0.Final.zip
+    # ln -s /opt/wildfly-34.0.0.Final /opt/wildfly
+
+== Standalone or Domain mode ?
+
+   # MODE=standalone
+
+== Create a wildfly user and group
+
+    # groupadd -r wildfly
+    # useradd -r -g wildfly -d /opt/wildfly -s /sbin/nologin wildfly
+
+== Configure systemd
+
+    # cd /opt/wildfly/bin/systemd
+    # cp wildfly-$MODE.conf /etc/sysconfig/
+    # cp wildfly-$MODE.service $(pkg-config systemd --variable=systemdsystemunitdir)
+    # chown -R wildfly:wildfly /opt/wildfly/
+    # chmod +x /opt/wildfly/bin/systemd/launch.sh
+    # systemctl daemon-reload
+
+=== If SELinux is enabled, you need to set the appropriate context for the launch.sh script:
+
+    # sudo chcon -R -t bin_t /opt/wildfly/
+    # sudo semanage fcontext -a -t systemd_unit_file_t '/opt/wildfly(/.*)?'
+
+== Start and enable WildFly
+
+    # systemctl start wildfly-$MODE.service
+    # systemctl enable wildfly-$MODE.service
+
+== Check the status of WildFly
+
+    # systemctl status wildfly-$MODE.service
+
+
+
+= How to remove WildFly as a systemd service
+
+== Standalone or Domain mode ?
+
+   # MODE=standalone
+
+== Stop and disable WildFly
+
+    # systemctl stop wildfly-$MODE.service
+    # systemctl disable wildfly-$MODE.service
+
+== If you are using SELinux, remove the custom context added
+
+    sudo semanage fcontext -d '/opt/wildfly(/.*)?'
+
+== Remove WildFly systemd service
+
+    # rm -f $(pkg-config systemd --variable=systemdsystemunitdir)/wildfly-$MODE.service
+    # rm -f /etc/sysconfig/wildfly-$MODE.conf
+    # systemctl daemon-reload
```

Comment:
Should we not add systemctl status wildfy and systemctl start wildfly with an example of the output?

Reply:
At this point, this is the section that describes how to remove the service.

The start / enable / status commands are described in the previous section. There is no sample of status output, though

Reply:
Somehow, I can't close this as resolved, but please do!